cypress/e2e/podcasts.cy.js

```diff
@@ -157,6 +157,25 @@ describe("newsdesk logged in", () => {
     cy.get("section[slotno=7] button[title='Delete']").click()
     cy.get("section[slotno=7]").should("not.contain", textToCopy)
   })
+
+  it("should dragging item in inbox", () => {
+    switchPodCast("dev sandbox")
+    cy.get("section[slotno=7] textarea").type("dragging item in inbox{enter}", {
+      force: true,
+    })
+    cy.get("section[slotno=7] button[title='Share to podcast']").click()
+    cy.get("section[slotno=7] input[id='dev'][type='checkbox']").click()
+    cy.get("section[slotno=7] textarea").type("dragging item{enter}", {
+      force: true,
+    })
+    cy.get("section[slotno=7]").should("contain", "dragging item")
+    cy.get("section[slotno=7] button[title='Share to podcast']").eq(0).click()
+    cy.get("section[slotno=7] input[id='dev'][type='checkbox']").click()
+    cy.get("#inbox-column").should("contain", "dragging item")
+    cy.get("#inbox-column ul li span div")
+      .eq(0)
+      .trigger("dragend", { clientX: 83, clientY: 153 }, { force: true })
```

Comment:
Please use selector rather than fixed xy positions

Reply:
I have used a selector for XY positions